package.json

```diff
@@ -18,23 +18,25 @@
   "tonicExampleFilename": "tonic-example.js",
   "scripts": {
     "clean": "rimraf dist",
-    "build": "npm run clean && npm run build:modules && npm run build:standalone",
+    "build": "npm run clean && npm run build:modules && npm run build:standalone && npm run build:types",
     "build:modules": "BABEL_ENV=modules babel lib -d dist/es-modules/",
     "build:standalone": "webpack && NODE_ENV=production webpack",
     "build:standalone:log": "NODE_ENV=production WEBPACK_MODE=log webpack --json --profile --progress > webpack-build-log.json && webpack-bundle-analyzer webpack-build-log.json",
-    "docs:build": "jsdoc -c jsdoc.json",
+    "build:types": "rimraf typings/generated && yarn tsc --declaration --noEmit false --emitDeclarationOnly true --allowJs false",
```

Comment:
Any reason we have a mix of `yarn` and `npm` ways of doing things?

Reply:
oops, a good catch. it's just a habit of using yarn 🤣